src/AudioIO.cpp

```diff
@@ -2398,8 +2408,14 @@ void AudioIO::StopStream()
   #endif
 
    if (mPortStreamV19) {
-      Pa_AbortStream( mPortStreamV19 );
+      // DV: Pa_CloseStream will close Pa_AbortStream internally,
+      // but it doesn't hurt to do it ourselves.
+      // PA_AbortStream will silently fail if stream is stopped.
+      if (!Pa_IsStreamStopped( mPortStreamV19 ))
+        Pa_AbortStream( mPortStreamV19 );
```

Comment:
I'm not sure why these changes (justified though they may be) are a necessary part of devendoring?  I'd like to understand whether they are really independent.